### ISSUE TITLE
v1.0.4 path setting problem

### ISSUE THREAD

**Issue by mizar** [nttld/setup-ndk#130]
setup-ndk seems to have a problem with the path setting in v1.0.4.

```
./main/script/android_build.sh: line 96: ndk-build: command not found
```

https://github.com/mizar/YaneuraOu/actions/runs/843952267

Since setup-ndk did not have this problem in v1.0.2, it seems to be a problem in v1.0.4.

https://github.com/mizar/YaneuraOu/actions/runs/844494958

**Comment by raftario** [comment]
Sorry for the inconvenience, I reset `v1` and `v1.0` to point to `v1.0.2` as a temporary fix until the underlying issue is found. I will also be updating the test suite to detect that error.

**Comment by raftario** [comment]
Closed by v1.0.5
